Cargo.toml

```diff
@@ -94,3 +96,18 @@ fuels-core = { version = "0.50.1", path = "./packages/fuels-core", default-featu
 fuels-macros = { version = "0.50.1", path = "./packages/fuels-macros", default-features = false }
 fuels-programs = { version = "0.50.1", path = "./packages/fuels-programs", default-features = false }
 fuels-test-helpers = { version = "0.50.1", path = "./packages/fuels-test-helpers", default-features = false }
+
+[patch.crates-io]
+fuel-asm = { git = "https://github.com/FuelLabs/fuel-vm/", branch = "master" }
+fuel-crypto = { git = "https://github.com/FuelLabs/fuel-vm/", branch = "master" }
+fuel-merkle = { git = "https://github.com/FuelLabs/fuel-vm/", branch = "master" }
+fuel-storage = { git = "https://github.com/FuelLabs/fuel-vm/", branch = "master" }
+fuel-tx = { git = "https://github.com/FuelLabs/fuel-vm/", branch = "master" }
+fuel-types = { git = "https://github.com/FuelLabs/fuel-vm/", branch = "master" }
+fuel-vm = { git = "https://github.com/FuelLabs/fuel-vm/", branch = "master" }
+fuel-core = { git = "https://github.com/FuelLabs/fuel-core", branch = "feature/transaction-policies" }
```

Comment:
It seems like this `fuel-core` branch has been merged; maybe we can point it to master for now?

Reply:
We point to the `rc` now.